prototype/templates/prototype/components/prototype_footer.html

```diff
@@ -0,0 +1,133 @@
+{% load static %}
+<footer id="footer">
+  <div class="container logo-container">
+    <img class="float-left" id="footer-dit-logo" src="{% static 'directory_components/images/dfit-logo-white.png' %}" alt="Department for International Trade"/>
+    <img class="float-right" id="footer-eig-logo" src="{% static 'directory_components/images/eig-logo-stacked.svg' %}" alt="Exporting is Great"/>
+  </div>
+  <nav class="container">
+    <div class="grid-row">
+      <div class="column-fifth">
+        <h2 class="heading heading-small" id="footer-advice-links">Advice</h2>
+        <ul aria-labelledby="footer-advice-links" class="footer-links-list">
+          <li>
+            <a href="#footer">How to start exporting</a>
```

Comment:
Once this prototype is accepted and goes live It'd be great to have IDs for links